### PR TITLE
Switching app engine tests to webtest and consolidating testing utils…

### PR DESCRIPTION
**Pull request by jonparrott**
… into AppEngineTestbedCase

### CODE REVIEW

**Comment by jonparrott** [comment]
Travis died on pep8, should be fixed. (man, import orders are annoying).

File: appengine/images/tests/test_guestbook.py [hunk 1]
```diff
@@ -66,30 +60,19 @@ def test_img(self):
...
 
         self.assertEqual(response.status_int, 200)
 
     def test_img_missing(self):
         # Bogus image id, should get error
-        request = webapp2.Request.blank('/img?img_id=123')
-        response = request.get_response(main.app)
-
-        self.assertEqual(response.status_int, 500)
+        self.app.get('/img?img_id=123', status=500)
```
Comment (waprin) [inline appengine/images/tests/test_guestbook.py]: Nothing to do with your commit, but we might want to use named constants for HTTP codes.

**Comment by waprin** [comment]
LGTM, one extraneous comment. I don't understand the benefits of switching to web request, maybe throw a blurb into the commit message?

File: appengine/images/tests/test_guestbook.py (hunk 1 above)
Comment (jonparrott) [inline appengine/images/tests/test_guestbook.py]: I do not feel strongly about this, but if you do, feel free to file a bug and we'll follow up with it.

**Comment by jonparrott** [comment]
Reduces the amount of code needed to test these samples, and will make it easier for us to reach full coverage later.

File: appengine/images/tests/test_guestbook.py (hunk 1 above)
Comment (waprin) [inline appengine/images/tests/test_guestbook.py]: Fair, oogleCloudPlatform/python-docs-samples/issues/85

**Pull request merged**